CHANGELOG.md

```diff
@@ -1,3 +1,84 @@
+## Release 0.16.0
+
+Highlights:
+
+* New network view to quickly ascertain what networks your containers belong to.
```

Comment:
It's not really a view, is it? @davkal how should we call this new features? "Networks on Canvas"?